openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx

```diff
@@ -216,6 +226,7 @@ const TourPage = () => {
         isTourRoute
         handleFeatureModal={handleCountChange}
         handleKeyDown={handleKeyDown}
+        handleOnClick={() => handleOnClick(searchValue)}
```

Comment:
No need to pass `searchValue` here, can use it directly like `handleKeyDown` method